Bindings/Java/Matlab/Utilities/plotMuscleFLCurves.m

```diff
@@ -188,25 +162,24 @@ function plotMuscleFLCurves(modelpath)
 
 end %function getMuscleCoordinates
 
-
-%% -----------------------------------------------------------------------------
 function [fl_active, fl_passive] = getForceLength(model, s, muscle)
-% This function gets the active and passive force--length values across the
-% possible fiber lengths of the muscle. fl_active and fl_passive are matrices
-% containing forces corresponding to each fiber length.
+% Get the active and passive force--length values across for the
+% possible fiber lengths of the muscle. fl_active and fl_passive are 
+% matrices containing forces corresponding to each fiber length.
```

Comment:
Usage here in the comments would be helpful. Its nice to say `help getForceLength` on matlab and see the input arguments and data returned. It isn't clear from the signature that *muscle* is a structure of some sort and not an `OpenSim::Muscle` reference.